functests/1_performance/cpu_management.go

```diff
@@ -380,3 +412,44 @@ func getStressPod(nodeName string) *corev1.Pod {
 		},
 	}
 }
+
+func getTestPodWithAnnotations(annotations map[string]string) *corev1.Pod {
```

Comment:
Should this be in utils/pods ?

Reply:
Hm, I don't think so.. here we creates specific pod used only in these tests..

Reply:
Ack